distributed/utils.py

```diff
@@ -1417,12 +1421,29 @@ def import_term(name: str) -> AnyType:
     return getattr(module, attr_name)
 
 
-async def offload(fn, *args, **kwargs):
+async def offload(  # type: ignore[valid-type]
```

Comment:
I mentioned this in another thread already. `offload` should not be used like this imo. offload is our backwards compatible version of `asyncio.to_thread` and I don't want us to use this terminology for every "run_in_executor". If you want to reuse this functionality, please copy the code / change the name of this function

Reply:
Done.